Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -15,6 +15,14 @@
 f = open(os.path.join(vcs.prefix,"share","vcs","wmo_symbols.json"))
 wmo = json.load(f)
 
+def isTrue(x):
+    """Test if an unsanitized user input is 'true'."""
+    # I'm just encapsulating existing logic here, but this seems to miss some
+    # other cases users may enter. What about:
+    #    - Non-zero integers other than 1?
+    #    - "yes", "Yes", "Y"?
+    return x in ("y", 1, True)
```

Comment:
@dlonie most attribute setting go through VCSValidationFunction which takes any of the has an input and sets it to True anyway. I know of one case where somehow VisTrails bypass this and sets directly the _attribute and we endup with "False" (string not boolean) but this wouldn't cover it anyway. I think we should remove this test, and if anything breaks, make sure the attribute is actually set via VCSValidation function.